src/Symfony/Bundle/FrameworkBundle/Resources/config/esi.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Resources\config;
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
+use Symfony\Component\HttpKernel\EventListener\SurrogateListener;
+use Symfony\Component\HttpKernel\HttpCache\Esi;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('esi', Esi::class)
+
+        ->set('esi_listener', SurrogateListener::class)
+            ->tag('kernel.event_subscriber')
```

Comment:
please put the `tag` after the `args` for consistency with other pull requests. The original order in the XML file is deprecated.

Reply:
thanks your comment.